packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.Basic.Example.tsx

```diff
@@ -22,6 +22,9 @@ export class HorizontalStackBasicExample extends React.Component<{}, {}> {
       },
       tenGapStack: {
         childrenGap: 10
+      },
+      tenPaddingStack: {
+        padding: 10
```

Comment:
nit: it'd be nicer if these had meaningful semantic names rather than enumerated values in text...